source/popover.txt

```diff
@@ -0,0 +1,41 @@
+Popovers
+========
+
+The GtkPopover is a seperate Window used for displaying additional information and are often used as 
+a part of button menus and context menus. Their uses are similar to those of dialog windows with the advantage of being less
+disruptive and having a connection with the widget the popover is pointing to.
+Popovers have are visually connected to a related Widget with a small triangle.
+
+
+A Popover can be created with :class:`Gtk.Popover`; for opening the Popover use :meth:`Gtk.Widget.show_all`.
+
+Custom Popover
+--------------
+
+A Widget can be added to a Popover using the :meth:`Gtk.Container.add`.
+
+Example
+^^^^^^^
+
+.. image:: ../images/popover_example.png
+
+.. literalinclude:: ../examples/popover_example.py
+    :linenos:
+
+Menu Popover
+------------
+
+A Popover can be created from a Menu using :meth:`Gtk.Popover.new_from_model` and can be changed after creation with :meth:`Gtk.Popover.bind_model`.
+
+Example
+^^^^^^^
+
+.. image:: ../images/popover_advanced_example.png
```

Comment:
The image is different from what I see when running the example. I get 3 items "Keyboard Shortcuts", "Help", and "About", but all of them are deactivated and cannot be interacted with.

Reply:
I reworked that example now. The items were inactive, because on actions were attached. I reworked that example and the actions are working now. (I'll probably also create documentation for the KeyboardShortcuts-Window, but as a seperate part of the docs)